prow/jobs/kyma-project/template-operator/template-operator.yaml

```
@@ -167,4 +163,97 @@ postsubmits: # runs on main
           - name: signify-secret
             secret:
               secretName: signify-dev-secret
-  
\ No newline at end of file
+    - name: release-template-operator-build
+      annotations:
+        description: "Job to build template-operator for a release."
+        owner: "jellyfish"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "release-template-operator-build"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-sa-kyma-push-images: "true"
+        preset-signify-prod-secret: "true"
+      always_run: true
+      skip_report: false
+      decorate: true
+      cluster: trusted-workload
+      max_concurrency: 10
+      branches:
+        - ^v?\d+\.\d+\.\d+(?:-.*)?$
+      spec:
+        containers:
+          - image: "eu.gcr.io/sap-kyma-neighbors-dev/image-builder:v20230313-8dfce5f0b-buildkit"
+            securityContext:
+              privileged: true
+              seccompProfile:
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            command:
+              - "sh"
+            args:
+              - "-c"
+              - "/image-builder --name=template-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=components/operator/Dockerfile --tag=$PULL_BASE_REF"
```

Any reason why it's written like that? image-builder is designed to use as entrypoint. If there are no reason, use it as entrypoint.
```suggestion
            command:
              - "/image-builder"
            args:
              - "--name=template-operator"
              - "--config=/config/kaniko-build-config.yaml"
              - "--context=."
              - "--dockerfile=components/operator/Dockerfile"
              - "--tag=$(PULL_BASE_REF)"
```